content/Proposals/Accepted/202205-oo.md

```diff
@@ -0,0 +1,63 @@
+## Evolution of the Observability Operator (fka MSO)
+
+* **Owners:**
+  * [`@dmohr`](https://github.com/danielm0hr)
+  * [`@jan--f`](https://github.com/jan--f)
+
+* **Related Tickets:**
+  * https://issues.redhat.com/browse/MON-2279
+  * https://issues.redhat.com/browse/MON-2482
+  * https://issues.redhat.com/browse/MON-2508 (short-term focus)
+
+* **Other docs:**
+  * [Original proposal for MSO](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/monitoring-stack-operator.md)
+
+> TL;DR: As a mid-to-long term vision for the Monitoring Stack Operator we propose to rename MSO as OO (Observability Operator). With the name, we propose to establish OO as *the* open source, (single) cluster-side component for OpenShift observability needs. In conjunction with Observatorium / RHOBS - as *the* multi-cluster, multi-tenant, scalable observability backend component. OO is thought to manage different kinds of cluster-side monitoring, alerting, logging, and tracing (and potentially profiling) stack setups covering the needs of OpenShift variants like the client-side for the fully managed multi-cluster use cases to HyperShift and single node air gapped setups.
+
+## Why
+
+With the rise of new OpenShift variants with very different needs regarding observability, the desire for a consistent way of providing differently configured monitoring stacks grows. Examples:
+- Traditional single-cluster on-prem OpenShift deployments need a self-contained monitoring stack where all components (scraping, storage, visualization, alerting, log collection) run on the same cluster. This is the kind of setup [Cluster Monitoring Opeator](https://github.com/openshift/cluster-monitoring-operator) was designed for.
+- Multi-cluster deployments need a central (aggregated, federated) view on metrics, logs and alerts. Certain components of the stack don't run on the workload clusters but in some central infrastructure.
+- Resource-constraint deployments need a stripped down version of the monitoring stack, e.g. only forwarding signals to a central infrastructure or disabling certain parts of the stack completely.
+- Mixed deployments (e.g. OpenShift + OpenStack) are conceptually very similar to the multi-cluster use case, also needing a central pane of glass for observability signals.
+- Special purpose deployments (e.g. OpenShift Data Foundation) have special requirements when it comes to monitoring, that are tricky to align with the existing CMO setup.
```

Comment:
would you have one or two examples to illustrate?

Reply:
@jan--f: This line is from you. Do you have an example?